themes/5ePhb.style.less

```diff
@@ -452,6 +452,11 @@ body {
 		.useColumns(0.96, @fillMode: balance);
 	}
 
+	//SRD Monster Blocks
+	.monster.srd p+p{
+		text-indent: 0em;
```

Comment:
don't need to specify units if value is zero

Reply:
If SRD monsters need special treatment here, it seems like we should just fix the original monster styling to match. We should be able to make them fully compatible so we don't have to maintain two different statblock styles.

Reply:
In the `monster` block generated by the existing snippet, each **Trait** and **Action** is separated by a `\n:\n`, whereas in the scraped monster data, the items are only separated by the closing and opening tags of a `<p>` block. So the styling fix is really only because of the limitations of the scraped data.
A little processing of the data - making **Traits**, **Actions**, and **Legendary Actions** into lists, for example, rather than just a block of HTML encoded text - would likely resolve this.